Classes/Controller/GetFileController.php

```diff
@@ -95,48 +99,74 @@ public function attachmentAction()
 
                     header('Content-Type: text/xml; charset=UTF-8');
 
-                }
+                    return $metsXml;
+
+                } else {
+
+                    http_response_code(404);
```

Comment:
Is there no solution to set response header with a custom message? I think this should be possible inside TYPO3.